engine/execution/state/delta/view_test.go

```diff
@@ -561,7 +573,9 @@ func TestView_Reads(t *testing.T) {
 	})
 }
 
-func hashIt(spock hash.Hasher, value []byte) error {
+func hashIt(spock hash.Hasher, value []byte) {
```

Comment:
I'd love a 1-liner code comment on when panicking hashers are expected (e.g. linkage problem) and how we chose to consider this unrecoverable.

Reply:
Yup I'll add a comment to justify this. In this case, it is only considered unrecoverable in the test.